src/core/libraries/save_data/save_memory.cpp

```diff
@@ -44,6 +44,7 @@ struct SlotData {
 
 static std::mutex g_slot_mtx;
 static std::unordered_map<u32, SlotData> g_attached_slots;
+static size_t g_memory_size = 0;
```

Comment:
Store this in SlotData struct instead of  global, games can have multiple save at the same time. You can store this during `SetupSaveMemory` call